server/api/v1alpha1/workspace_types.go

```diff
@@ -27,6 +27,9 @@ const (
 	WorkspaceVisibilityCommunity WorkspaceVisibility = "community"
 	// WorkspaceVisibilityPrivate Private value for Workspaces visibility
 	WorkspaceVisibilityPrivate WorkspaceVisibility = "private"
+
+	// LabelInternalDomain if the requesting user is the owner of the workspace
+	LabelIsOwner string = "internal.workspaces.konflux-ci.dev/is-owner"
```

Comment:
I'd suggest to use the constant in `workspacesv1alpha1 "github.com/konflux-workspaces/workspaces/operator/api/v1alpha1"`

```suggestion
	LabelIsOwner string = workspacesv1alpha1.LabelInternalDomain + "is-owner"
```